src/test/java/uk/gov/pay/api/service/GetPaymentServiceTest.java

```diff
@@ -100,5 +102,17 @@ public void testGetPaymentWithCorporateCardSurcharge() {
         CardPayment payment = (CardPayment) paymentResponse.getPayment();
         assertThat(payment.getCorporateCardSurcharge().get(), is(250L));
         assertThat(payment.getTotalAmount().get(), is(2250L));
+        assertThat(paymentResponse.getLinks().getCapture(), is(nullValue()));
+    }
+
+    @Test
+    @PactVerification({"connector"})
+    @Pacts(pacts = {"publicapi-connector-get-payment-with-awaiting_capture_request_state"})
```

Comment:
Not sure about this mixing dashes and underscores in the name

Reply:
my bad!